releases/v2.8.1.toml

```diff
@@ -18,7 +18,7 @@ See changelog below for full list of changes.
 
 ### CI
 * ci: use proper git ref for versioning [#3595](https://github.com/distribution/distribution/pull/3595)
-* Go: make Go version explicit and pin it to the latest 1.16 release [3604](https://github.com/distribution/distribution/pull/3604)
+* Go: make Go version explicit and pin it to the latest 1.18 release [3604](https://github.com/distribution/distribution/pull/3604)
```

Comment:
This should be reverted, as it's updating changes for the v2.8.1 release (which did use go1.16)

Reply:
reverted.